src/ARAnchorManager.js

```diff
@@ -0,0 +1,122 @@
+import { Object3D, Vector3, Matrix4, EventDispatcher } from 'three';
+
+/**
+ * A class to manage registration and unregistration of anchors
+ * for any THREE.Object3D. Anchors are world space and handle updating
+ * the position of the object to which one is assigned.
+ * AnchorManager emits events pertaining to anchors.
+ */
+export default class ARAnchorManager extends EventDispatcher {
+  /**
+  * @param {VRDisplay} vrDisplay 
+  */
+  constructor(vrDisplay) {
+    super();
+    if (!(vrDisplay instanceof window.VRDisplay) || !vrDisplay.getAnchors) {
+      throw new Error('invalid vr display parameter');
```

Comment:
Maybe a second/separate check for `vrDisplay.getAnchors` so we can error out a more accurate/help message: "VRDisplay does not support anchors. Upgrade to latest AR browser to get anchor support: <link>"

Reply:
Fixed.